src/ScottPlot/Plottable/FinancePlot.cs

```diff
@@ -28,12 +28,12 @@ public class FinancePlot : IPlottable
         /// <summary>
         /// If True then OHLC timestamps are ignored and candles are placed at x= 0, 1, 2, 3, etc.
         /// </summary>
-        public bool Sqeuential;
+        public bool Sequential;
```

Comment:
Renaming public field/property - breaking change

Reply:
Thank you for identifying this issue! I'll track its discussion in #705

Reply:
Thanks @jl0pd! This is a breaking change, but will be released with 4.1-beta which has other breaking changes so this is an excellent time to fix it. I corrected this in c3e1062fe1f8a07a3e3519e9cdf62f40ac155cd4